python/cudf/cudf/_libxx/cpp/utilities/nvtx_utils.pxd

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from libc.stdint cimport uint32_t
+
+ctypedef uint32_t underlying_type_t_color
```

Comment:
This shouldn't be here as it's not an externed definition. It should go in `_libxx/nvtx.pxd`

Reply:
Sorry. I was following the convention from types. I felt like it was a bit weird. Fixing.

Reply:
This is still in the wrong header